pom.xml

```diff
@@ -168,28 +168,47 @@
           </excludes>
         </configuration>
       </plugin>
+      <!--      <plugin>-->
+      <!--        <groupId>org.apache.maven.plugins</groupId>-->
+      <!--        <artifactId>maven-surefire-plugin</artifactId>-->
```

Comment:
don't need surefire anymore?

Reply:
That is what I am unsure about, I know that it runs unit tests and generate reports, which are usually text and XML files, Are they useful for us in any way? 

I added the same comment in this PR summary, I personally believe that having a JACOCO plugin that could give us interactive HTML code coverage is very useful.

Reply:
that is why I just commented it so that we can discuss and see what others think.

Reply:
ah, I missed your description. I'm not sure either. Maybe our java gurus have some ideas Joe, Nate, Cicilia, or Greg